build/run-wasm-tests.sh

```diff
@@ -11,10 +11,11 @@ function interrupt {
 trap interrupt SIGINT SIGTERM
 
 # Generate the test tarball from the asset files.
-go run test/wasm/cmd/testgen.go \
-   --input-dir test/wasm/assets \
-   --output _test/testcases.tar.gz \
-   $@
+if [ ! -f _test/testcases.tar.gz ]; then
```

Comment:
Since the test case tarball isn't getting re-created here, should we remove it in the `wasm-test` target in the Makefile?